spec.bs

```diff
@@ -973,7 +973,11 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       |config|["{{AuctionAdConfig/additionalBids}}"] until an associated [=request=] whose

```

Comment:
I'm wondering if someone else should review too? I don't have much experience with specs that integrate with the HTML DOM.